frontend/packages/dev-console/src/utils/__tests__/pipeline-test-data.ts

```diff
@@ -12,7 +12,7 @@ export const mockPipelinesJSON: K8sResourceKind[] = [
       params: [
         {
           name: 'APP_NAME',
-          type: 'string',
```

Comment:
`type` should be allowed 🤔  Why did you remove this?

Reply:
Ah because it's not part of the type 😞 

Can you just add it to the type instead? 
It's [supported by upstream](https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md#specifying-parameters)
> Each declared parameter has a type field, which can be set to either array or string.

We'll want the type to probably be:

```
  type: 'string' | 'array';
```
Instead of a straight `string` type... this way we can easily see what we support.